_assets/css/styles.min.css

```diff
@@ -1 +1 @@
.2rem;margin:4rem 0;border-bottom:1px solid #E2E2E2}article #tags ul{list-style:none}article #tags li{display:inline-block;border:1px solid #15357F;border-radius:4px;padding:4px 8px;margin-right:8px;font-weight:400;font-size:12px;line-height:16px}article #tags li:hover,article #tags li:focus{background:#E6E9F3}article #tags a{color:inherit;text-decoration:none}.btn{display:inline-block;text-decoration:none;border:none;font-size:1.4rem;line-height:1.6rem;text-align:center;border-radius:.4rem;position:relative;cursor:pointer;font-weight:500;color:#0B132A;padding:.8rem 1.2rem}.btn:focus{outline:0}.btn-orange{background:#F5821F;color:#FFFFFF !important;border:none}.btn-orange:before{background:linear-gradient(45deg, #F5821F, #FFFFFF, #F5821F)}.btn-orange-secondary:hover,.btn-orange-secondary:focus{background-color:#BA6622}.btn-bordered{border:1px solid #ACADAE}.btn-transparent{font-size:1.5rem;font-weight:700;text-align:left;color:#0B132A;font-weight:bold;background:none;padding-left:0}.btn-transparent img{vertical-align:middle;margin:0 1rem}.btn-transparent:hover,.btn-transparent:focus{color:#4F5665}.btn-white{color:#FFFFFF}.btn-white:hover,.btn-white:focus{color:#A7AAB2 !important}code .hll{background-color:#ffffcc}code .c{color:#408080;font-style:italic}code .err{border:1px solid #FF0000;border:none}code .k{color:#008000;font-weight:bold}code .o{color:#666666}code .cm{color:#408080;font-style:italic}code .cp{color:#BC7A00}code .c1{color:#408080;font-style:italic}code .cs{color:#408080;font-style:italic}code .gd{color:#A00000}code .ge{font-style:italic}code .gr{color:#FF0000}code .gh{color:#000080;font-weight:bold}code .gi{color:#00A000}code .go{color:#888888}code .gp{color:#000080;font-weight:bold}code .gs{font-weight:bold}code .gu{color:#800080;font-weight:bold}code .gt{color:#0044DD}code .kc{color:#008000;font-weight:bold}code .kd{color:#008000;font-weight:bold}code .kn{color:#008000;font-weight:bold}code .kp{color:#008000}code .kr{color:#008000;font-weight:bold}code .kt{color:#B00040}code .m{color:#666666}code .s{color:#BA2121}code .na{color:#7D9029}code .nb{color:#008000}code .nc{color:#0000FF;font-weight:bold}code .no{color:#880000}code .nd{color:#AA22FF}code .ni{color:#999999;font-weight:bold}code .ne{color:#D2413A;font-weight:bold}code .nf{color:#0000FF}code .nl{color:#A0A000}code .nn{color:#0000FF;font-weight:bold}code .nt{color:#008000;font-weight:bold}code .nv{color:#19177C}code .ow{color:#AA22FF;font-weight:bold}code .w{color:#bbbbbb}code .mf{color:#666666}code .mh{color:#666666}code .mi{color:#666666}code .mo{color:#666666}code .sb{color:#BA2121}code .sc{color:#BA2121}code .sd{color:#BA2121;font-style:italic}code .s2{color:#BA2121}code .se{color:#BB6622;font-weight:bold}code .sh{color:#BA2121}code .si{color:#BB6688;font-weight:bold}code .sx{color:#008000}code .sr{color:#BB6688}code .s1{color:#BA2121}code .ss{color:#19177C}code .bp{color:#008000}code .vc{color:#19177C}code .vg{color:#19177C}code .vi{color:#19177C}code .il{color:#666666}
```

Comment:
Should we pull from remote source for this or should we have it locally?

Reply:
In slack Igor said he would amend this PR to bring that remote resource in locally.

Reply:
done